README.md

```diff
@@ -97,6 +97,9 @@ For more details about how Cruise Control achieves that, see
 To read more about the configurations. Check the 
 [configurations wiki page](https://github.com/linkedin/cruise-control/wiki/Configurations).
 
+### Artifact ###
+Published at [Jfrog Artifactory](https://linkedin.jfrog.io/linkedin/webapp/#/artifacts/browse/tree/General/cruise-control). Latest release version is 0.1.4
```

Comment:
Should we avoid hardcoding the latest release version in `README`? Instead we may say `See [available releases](https://github.com/linkedin/cruise-control/releases).`